src/generator.cs

```diff
@@ -1117,6 +1127,176 @@ static string FindProtocolInterface (Type type, MemberInfo pi)
 		return "I" + type.Name;
 	}
 
+	// TODO: Alex
```

Comment:
Yes, I still need to cleanup comments, I use that as a navigation token on the file, I wish XS supported opening the same file multiple times